gitlab-ci/tools/repro-check.sh

```diff
@@ -249,7 +249,7 @@ if [ -n "$proposal_id" ]; then
 
     log "Check the proposal url is correctly formatted"
     if [ "${guestos_proposal:-}" == "true" ]; then
-        expected_url="https://download.dfinity.systems/ic/$git_hash/guest-os/update-img/update-img.tar.gz"
+        expected_url="https://download.dfinity.systems/ic/$git_hash/guest-os/update-img/update-img.tar.zst"
     else
         expected_url="https://download.dfinity.systems/ic/$git_hash/host-os/update-img/update-img.tar.gz"
```

Comment:
We can update both guestos and hostos urls to zst! All gz references in the script can be updated to zst 👍